src/platform/Zephyr/WiFiManager.cpp

```diff
@@ -60,19 +51,46 @@ in6_addr ToZephyrAddr(const Inet::IPAddress & address)
     return zephyrAddr;
 }
 
-net_if * GetInterface(Inet::InterfaceId interfaceId = Inet::InterfaceId::Null())
+net_if * GetInterface(Inet::InterfaceId ifaceId = Inet::InterfaceId::Null())
 {
-    return interfaceId.IsPresent() ? net_if_get_by_index(interfaceId.GetPlatformInterface()) : net_if_get_default();
+    return ifaceId.IsPresent() ? net_if_get_by_index(ifaceId.GetPlatformInterface()) : net_if_get_default();
 }
 
 } // namespace
 
+// These enums shall reflect the overall ordered disconnected->connected flow
+// NOTE: it's NOT a unique keyed map
+const WiFiManager::StatusMap::StatusPair WiFiManager::StatusMap::sStatusMap[] = {
```

Comment:
Just a question.. why this WPA stuff is in Zephyr and not nRF Connect platform?

Reply:
Are we planning to contribute WPA supplicant to clear Zephyr?

Reply:
Hmm maybe I haven't got the clear concept on how nrfconnect platform code is organized but I thought that WiFiManager was something similar to other Zephyr specific implementations like KeyValueStoreManagerImpl, NFCManagerImpl or ThreadStackManagerImpl, and should be used in similar fashion as such. And regarding wpa_supplicant, I am not sure what kind of contribution you mean, but we are already using the Zephyr-port of the wpa_supplicant.

Reply:
Let's merge this PR and move the discussion to some other channel. I hope it's ok. I case any changes are requested I will do it in separate PR.